process-compose/suzuka-full-node/process-compose.telemetry.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+
```

Comment:
If you can add an OTLP runner, that would be great.

Reply:
Our nix flake does package in Docker, so you can just spawn the container from process compose.

Reply:
```
just suzuka-full-node native build.setup.eth-local.celestia-local.test -t=false
```